server/src/server.ts

```diff
@@ -585,20 +581,19 @@ const createServer = async () => {
 
   let server;
 
-  console.log("process.env")
-  console.log(process.env)
+  console.log('process.env');
```

Comment:
Could you remove these console.logs while you're at it

Reply:
Done!